backend/server.js

```diff
@@ -2,14 +2,41 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import crypto from 'crypto'
+import bcrypt from 'bcrypt'
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/authAPI"
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/auth"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
+const User = mongoose.model('User', {
+  email:{
+    type: String,
+    unique: true,
+    required: true
+  },
+  password:{
+    type: String,
+    required: true,
+    minLength: 8
+  },
+  accessToken:{
+    type: String,
+    default: () => crypto.randomBytes(128).toString('hex')
+  }
+})
+
```

Comment:
Nice and clean model! Consider to update the model with maxLength: 12, since it's written in the frontend.